### DISCUSSION TITLE
error with triangulate

### DISCUSSION THREAD

**Discussion by Marwa-AL** [cityjson/cjio#173] [Q&A]
I am trying to triangulate cityjson file but facing the following problem: 

I have both cityjson files 1.0 and 1.1, with 1.0 I don't get the message below but the file doesn't get triangulated 
with 1.1 I am getting the message below and the file is not triangulated too. 

C:\Users\00\AppData\Local\Programs\Python\Python39\lib\site-packages\cjio\geom_help.py:43: RuntimeWarning: overflow encountered in scalar multiply
  n[1] += ( (poly[i][2] - poly[ne][2]) * (poly[i][0] + poly[ne][0]) )
C:\Users\00\AppData\Local\Programs\Python\Python39\lib\site-packages\cjio\geom_help.py:44: RuntimeWarning: overflow encountered in scalar multiply
  n[2] += ( (poly[i][0] - poly[ne][0]) * (poly[i][1] + poly[ne][1]) )
C:\Users\00\AppData\Local\Programs\Python\Python39\lib\site-packages\cjio\geom_help.py:42: RuntimeWarning: overflow encountered in scalar multiply
  n[0] += ( (poly[i][1] - poly[ne][1]) * (poly[i][2] + poly[ne][2]) )

**Comment by hugoledoux** [comment]
If your file have geometric errors (duplicate points, self-intersections, etc.) it's possible that the triangulation will fail.

try adding the `--sloppy` flag when triangulating, that should make it finish (but whether the triangulation is valid this is another issue...).

```
cjio myfile.city.json triangulate --sloppy save mytriangles.city.json
```
